src/Capability/Registry/ReferenceHandler.php

```diff
@@ -85,10 +85,28 @@ private function prepareArguments(\ReflectionFunctionAbstract $reflection, array
         $finalArgs = [];
 
         foreach ($reflection->getParameters() as $parameter) {
-            // TODO: Handle variadic parameters.
             $paramName = $parameter->getName();
             $paramPosition = $parameter->getPosition();
 
+            // Handle variadic parameters
```

Comment:
i think this is clear with the next line of code already and we can remove this comment
```suggestion
```